application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -148,29 +153,15 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
         OptionMapping maxUsesOption = event.getOption(MAX_USES_OPTION);
         OptionMapping maxAgeOption = event.getOption(MAX_AGE_OPTION);
 
+        String applicationId;
+        String applicationName;
```

Comment:
Dont declare variables before they are needed. Put this as late as possible, i.e.  at line 165.

Reply:
Moved these now.